src/main/java/org/isf/vactype/manager/VaccineTypeBrowserManager.java

```diff
@@ -62,23 +62,23 @@ protected void validateVaccineType(VaccineType vaccineType, boolean insert) thro
 		List<OHExceptionMessage> errors = new ArrayList<>();
 		if (key.isEmpty()) {
 			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.common.error.title"),
-					MessageBundle.getMessage("angal.vactype.pleaseinsertacode"),
+					MessageBundle.getMessage("angal.common.pleaseinsertacode.msg"),
 					OHSeverityLevel.ERROR));
 		}
 		if (key.length() > 1) {
 			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.common.error.title"),
-					MessageBundle.getMessage("angal.vactype.codemaxchars"),
+					MessageBundle.getMessage("angal.common.thecodeistoolongmax1char.msg"),
 					OHSeverityLevel.ERROR));
 		}
```

Comment:
should we use a format mask here?

Reply:
Didn't change this one because of "chars" vs "char" as noted above.